test/app-location.html

```diff
@@ -12,13 +12,8 @@
 <head>
   <title>app-location</title>
 
-  <script src="../../../@webcomponents/webcomponentsjs/webcomponents-bundle.js"></script>
-  <script src="../../../wct-browser-legacy/browser.js"></script>
-  <script src="../../../url/url.js"></script>
-
-  <script type="module" src="../../polymer/polymer-legacy.js"></script>
-  <script type="module" src="../../iron-test-helpers/mock-interactions.js"></script>
-  <script type="module" src="../app-location.js"></script>
+  <script src="../node_modules/@webcomponents/webcomponentsjs/webcomponents-bundle.js"></script>
```

Comment:
Im assuming most of these test diffs are whitespace right?

Reply:
yup. See this link: